load-org-data.sh

```diff
@@ -10,5 +10,7 @@ git config core.sparseCheckout true
 
 echo "data/" >> .git/info/sparse-checkout
 echo "contributors/" >> .git/info/sparse-checkout
+echo "colorPalette.json" >> .git/info/sparse-checkout
+
+git pull origin main
```

Comment:
Why was this added?

```suggestion
```

Reply:
git pull origin main is already in .sh file

Reply:
oh sorry, my bad